### DISCUSSION TITLE
Filter relation excluding many to many table

### DISCUSSION THREAD

**Discussion by tonytoth** [TriPSs/nestjs-query#104] [General]
@TriPSs I have a feature that sounds like this.
Give me all products that are not hidden. (hidden meaning a many to many table userId productId)
Im not sure we can achieve this using the current filter system.

**Comment by TriPSs** [comment]
Hmm, don't think this is possible with the current filters indeed, you could create a custom service that applies that filter the the query builder?

**Reply by tonytoth** [reply]
I ended up using a query in a custom authorizer.
